eXo/Sources/Controllers/On-Boarding/ServerSelectionViewController.swift

```diff
@@ -28,25 +28,59 @@ class ServerSelectionViewController: UIViewController {
     @IBOutlet weak var mostRecentServerLabel: UILabel!
     var defaultServer:Server?
     
+    @IBOutlet weak var defaultServerButton: UIButton!
+    @IBOutlet weak var addServerButton: UIButton!
     
+    @IBOutlet weak var discovereXoTribeButton: UIButton!
+    
+    @IBOutlet weak var eXoPlatformDescriptionLabel: UILabel!
     // MARK: View Controller lifecycle
     
-
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        Tool.applyBorderForView(defaultServerButton)
+        Tool.applyBorderForView(addServerButton)
+        eXoPlatformDescriptionLabel.text = NSLocalizedString("OnBoarding.Title.eXoPlatformDescription", comment: "")
+    }
+    
     override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(animated)
         if (ServerManager.sharedInstance.serverList.count > 0){
             defaultServer = ServerManager.sharedInstance.serverList.firstObject as? Server
+            defaultServerButton.setTitle(defaultServer?.serverURL.stringURLWithoutProtocol(), forState: .Normal)
         } else {
             defaultServer = Server(serverURL: Config.communityURL)
         }
-        // Do not show the protocol to save place
-        mostRecentServerLabel.text = (defaultServer?.serverURL)!.stringURLWithoutProtocol()
-
+        
+        self.setButtonsTitle()
+        
         // the navigation controller is alway shown in this screen
-        self.navigationController?.navigationBarHidden = false
-        self.navigationController?.topViewController?.title = NSLocalizedString("OnBoarding.Title.SignInToeXo", comment:"")
+        self.navigationController?.navigationBarHidden = true
     }
     
+    func setButtonsTitle () {
+        /*
+        Set the buttons titles
+        */
+        if (defaultServer?.serverURL == Config.communityURL) {
+            defaultServerButton.setTitle(NSLocalizedString("OnBoarding.Title.DiscovereXo", comment: ""), forState: .Normal)
```

Comment:
On this screen, we display the shortened URL even for community => remove this condition.